packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -251,7 +252,7 @@ export function checkForUnmatchedText(
     }
   }
 
-  if (isConcurrentMode && enableClientRenderFallbackOnHydrationMismatch) {
+  if (isConcurrentMode && enableClientRenderFallbackOnTextMismatch) {
```

Comment:
Should this be `enableClientRenderFallbackOnHydrationMismatch && enableClientRenderFallbackOnTextMismatch`?

Reply:
we'll keep them separate